src/components/app/App.tsx

```diff
@@ -1,30 +1,29 @@
-import bugsnag, { Bugsnag } from "@bugsnag/js";
+import * as React from "react";
+import bugsnag from "@bugsnag/js";
 import bugsnagReact from "@bugsnag/plugin-react";
-import {
-  AppBar,
-  createMuiTheme,
-  IconButton,
-  Link,
-  Menu,
-  MenuItem,
-  Toolbar,
-  Typography
-} from "@material-ui/core";
-import * as MenuIcon from "@material-ui/icons/Menu";
+
+// Bugsnag docs recommend instantiating the BugsnagClient as soon as possible and before the rest of the module's imports
+const bugsnagClient = bugsnag({
+  apiKey: "2d77ffd5445ab337cd3c996a2400438a",
```

Comment:
The key should be added as a field to the config and read from there

Reply:
the reason i did it this way is because bugsnag recommends instantiating the Client as soon as possible, i.e. before component mounts and therefore before a request would be made to fetch the config details. What is the significance of fetching the api key from the config file as opposed to hard coding it?

Reply:
Since we're making an open source product, we can't be hardcoding things specific to us. I think it's fine to load this after fetching the config, because if there's a problem with that the entire app is unusable anyway

Reply:
Also, this hardcoded api key will be replaced by `process.env.BUGSNAP_API_KEY` once I use `env-cmd` to set env

Reply:
gotcha! updating now.

Reply:
api keys like this should be accessed through environment variables, which is not possible with the way things currently are, i.e. fetches and uses the config file. since the api key is public and i will be removing the hardcoded value in my next PR to use `env-cmd` to set environment variables, is it possible to leave as is for now?

Reply:
Let's leave it blank for now and not deploy until both are in. Even though it's not a secret I'd like to avoid committing that string since there might be web crawlers looking for long hex strings

Reply:
Sounds good to me. Unfortunately tho can't leave it blank because bugsnag complains.

Reply:
what I'll do is replace the bugsnag error boundary with a React implementation fo ErrorBoundary until the `env-cmd` ticket is done